### PR TITLE
Metrics: add LDAP active sync summary metric

### PR DESCRIPTION
**Pull request by markelog**

### CODE REVIEW

File: pkg/infra/metrics/metrics.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ var (
 	M_Aws_CloudWatch_GetMetricData       prometheus.Counter
 	M_DB_DataSource_QueryById            prometheus.Counter
 
+	// LDAPUsersSyncExecutionTime is a metric for
+	// how much time it took to sync the LDAP users
+	LDAPUsersSyncExecutionTime prometheus.Summary
```
Comment (markelog) [inline pkg/infra/metrics/metrics.go]: With latest commits it seems we started to move away from snake case in favor of camel case

Comment (alexanderzobnin) [reply pkg/infra/metrics/metrics.go]: Yes, looks like.

**Review by alexanderzobnin — CHANGES_REQUESTED** [review]
Hm... I don't see where's metrics is collecting. Look at this example:
https://github.com/grafana/grafana/blob/ca628832ab913f0a79928a37b8204aa4e19a244e/pkg/services/alerting/eval_handler.go#L73

**Review by alexanderzobnin — APPROVED** [review]
Ah, sorry, it's in another repo. Now I see that.

File: pkg/infra/metrics/metrics.go (hunk 1 above)
Comment (torkelo) [reply pkg/infra/metrics/metrics.go]: yea, looks like it. Should create a chore/refactor issue to rename the others, looks a bit messy with two naming ways

**Comment by torkelo** [comment]
There is no milestone , should this not be in stable? Then it needs a milestone and cherry pick label. 

Also branch is not removed :)

**Comment by markelog** [comment]
> There is no milestone , should this not be in stable? Then it needs a milestone and cherry pick label.

Was about to do that :)

> Also branch is not removed :)

I re-used it in https://github.com/grafana/grafana/pull/18111, there is other undeleted branches too tho. Going hectically right now, until I understand the process more intimately

**Pull request merged**